snuba/admin/views.py

```diff
@@ -708,3 +711,109 @@ def snql_to_sql() -> Response:
             400,
             {"Content-Type": "application/json"},
         )
+
+
+@application.route("/allocation_policies")
+@check_tool_perms(tools=[AdminTools.CAPACITY_MANAGEMENT])
+def allocation_policies() -> Response:
+    return Response(
+        json.dumps(get_allocation_policies()), 200, {"Content-Type": "application/json"}
+    )
+
+
+@application.route("/allocation_policy_configs/<path:storage>", methods=["GET"])
+@check_tool_perms(tools=[AdminTools.CAPACITY_MANAGEMENT])
+def get_allocation_policy_configs(storage: str) -> Response:
```

Comment:
is this the current configuration values or what can be configured on the policy?

Reply:
This gets the current configuration values, I'll update the name to be more clear.